lib/Runtime/Types/ActivationObject.cpp

```diff
@@ -257,24 +257,24 @@ namespace Js
         }
     }
 
-    BOOL ActivationObjectEx::GetProperty(Var originalInstance, PropertyId propertyId, Var *value, PropertyValueInfo *info, ScriptContext *requestContext)
+    PropertyQueryFlags ActivationObjectEx::GetPropertyQuery(Var originalInstance, PropertyId propertyId, Var *value, PropertyValueInfo *info, ScriptContext *requestContext)
     {
-        if (__super::GetProperty(originalInstance, propertyId, value, info, requestContext))
+        if (JavascriptConversion::PropertyQueryFlagsToBoolean(__super::GetPropertyQuery(originalInstance, propertyId, value, info, requestContext)))
         {
             GetPropertyCore(info, requestContext);
-            return TRUE;
+            return Property_Found;
         }
-        return FALSE;
+        return Property_NotFound;
```

Comment:
No, because TypedArrayBase is not in ActivationObjectEx's inheritance chain.

---
In reply to: [106297091](https://github.com/Microsoft/ChakraCore/pull/2640#discussion_r106297091) [](ancestors = 106297091)